src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -148,9 +148,10 @@ void setActiveIndexer(LogstashIndexer<?> activeIndexer)
 
   @SuppressWarnings("deprecation")
   @Initializer(after = InitMilestone.JOB_LOADED)
-  public void migrateData()
+  public static void migrateData()
```

Comment:
actually what is the reason for making this method static?

Reply:
The initializers are always static. The code won't even compile in newer versions otherwise IIRC